### PR TITLE
gui, modules/SceCommonDialog: Update common strings and lang

### PR DESCRIPTION
**Pull request by Croden1999**
# About
- modules/SceCommonDialog: Update common strings and lang
- Adjust sort and corrected pt/zh/ch
Not all strings, the "Error Code" lang used error history of psvita settings. 

# Result
PS Vita

Vita3K
image
image

### CODE REVIEW

File: lang/tr.xml [hunk 1]
```diff
@@ -77,13 +77,17 @@ depending on its size and your hardware.</app_delete_note>
...
+Hata Kodu: {}</an_error_occurred>
 		<cancel>İptal</cancel>
 		<delete>Sil</delete>
 		<file_corrupted>Dosya bozuk.</file_corrupted>
+		<microphone_disabled>Enable the microphone.</microphone_disabled>
 		<no>Hayır</no>
 		<select_all>Tümünü Seç</select_all>
 		<select>Seç</select>
+		<submit>Submit</submit>
+		<ok>Tamam</ok>
```
Comment (Croden1999) [inline lang/tr.xml:90]: it is speical for here.

**Comment by Zangetsu38** [comment]
for OK, is no same for all languague ?

**Comment by Zangetsu38** [comment]
> 



> > for OK, is no same for all languague ?
> 
> only turkish is different on psvita, other languages are the same.

ok 🗿

**Review by Zangetsu38 — APPROVED** [review]

**Comment by Croden1999** [comment]
no, i not ready to merge. 😭

**Pull request merged**